eng/common/scripts/Helpers/ApiView-Helpers.ps1

```diff
@@ -19,18 +17,28 @@ function Map-LanguageToAPiview-Filter($language)
     return $lang
 }
 
-function Check-ApiReviewStatus($packageName, $language)
+function Check-ApiReviewStatus($packageName, $packageVersion, $language)
 {
   # Get API view URL and API Key to check status
-  Login-AzAccount
-  $url = Get-AzKeyvaultSecret -VaultName "AzureSDKPrepRelease-KV" -Name APIURL -AsPlainText
-  $apiKey = Get-AzKeyvaultSecret -VaultName "AzureSDKPrepRelease-KV" -Name APIKEY -AsPlainText
-
+  Write-Host "Checking API review status"
+  try
+  {
+    az login -o none
```

Comment:
We should avoid blindly calling 'az login' as it will always prompt. See the pattern at https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/scripts/Update-DevOps-Release-WorkItem.ps1#L26. Also as called out below we should try and avoid doing this in the helper functions but instead should do it in the entry point scripts.

Reply:
done